app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayViewModel.kt

```diff
@@ -26,4 +36,17 @@ class NavigationOverlayViewModel(
     val viewIsSplit: LiveData<Boolean> = sessionRepo.legacyState.map {
         it.currentUrl != URLs.APP_URL_HOME
     }
+
+    val pinnedTiles: Observable<ChannelDetails> = pinnedTileRepo.pinnedTiles
+            .observeOn(Schedulers.computation())
+            .map { it.values.map { it.toChannelTile(imageUtilityWrapper, formattedDomainWrapper) } }
+            .map { ChannelDetails(title = "Pinned Tiles", tileList = it) } // TODO extract string
```

Comment:
We'll need a new string for this actually, we should run this by Meridel (or her stand-in).

Reply:
I put a TODO about that and _still_ forgot.  Good catch!